Extension/src/LanguageServer/client.ts

```diff
@@ -3504,12 +3504,28 @@ export class DefaultClient implements Client {
                     character: range.end.character,
                     line: range.end.line
                 }
-            }
+            },
+            copyToClipboard: false
         };
 
+        if (copy) {
+            params.copyToClipboard = true;
+        }
+
         const result: CreateDeclarationOrDefinitionResult = await this.languageClient.sendRequest(CreateDeclarationOrDefinitionRequest, params);
         // TODO: return specific errors info in result.
-        if (result.changes === undefined) {
```

Comment:
When a 'request' is cancelled using a standard LSP cancelation, the resulting object will contain an error code, and will not be the expected object (so will not have the expected field(s)).  This check if an expected field was undefined handled that scenario. 

For example, in `codeActionProvider.ts`, line 66, you'll see that it's actually triggering a cancellation error exception (valid from providers) when this happens.

In this case, I'd recommend you retain a check such as:

```
if (result.edit === undefined)
   return;
```

To handle the possibility that the LSP request may have been canceled, and the expected object not actually returned.